From 5f21f3055a32878c0c9d9ebbbafef72d60a1ff49 Mon Sep 17 00:00:00 2001
From: Shunyong Yang <shunyong.yang@hxt-semitech.com>
Date: Mon, 1 Apr 2019 16:26:35 +0800
Subject: [PATCH] ACPI: property: restore _DSD data subnodes GUID comment
Git-commit: 5f21f3055a32878c0c9d9ebbbafef72d60a1ff49
Patch-mainline: v5.2-rc1
References: bsc#1111666

Commit 5f5e4890d57a ("ACPI / property: Allow multiple property
compatible _DSD entries") removed the comment of _DSD data subnodes
GUID.  Restore it.

Fixes: 5f5e4890d57a ("ACPI / property: Allow multiple property compatible _DSD entries")
Signed-off-by: Shunyong Yang <shunyong.yang@hxt-semitech.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/acpi/property.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/acpi/property.c
+++ b/drivers/acpi/property.c
@@ -33,6 +33,7 @@ static const guid_t prp_guids[] = {
 		  0x90, 0xe1, 0x92, 0x7a, 0x4e, 0x0c, 0x55, 0xa4),
 };
 
+/* ACPI _DSD data subnodes GUID: dbb8e3e6-5886-4ba6-8795-1319f52a966b */
 static const guid_t ads_guid =
 	GUID_INIT(0xdbb8e3e6, 0x5886, 0x4ba6,
 		  0x87, 0x95, 0x13, 0x19, 0xf5, 0x2a, 0x96, 0x6b);
